src/lib/client.js

```diff
@@ -178,13 +175,11 @@ class Client extends EventEmitter {
       return Promise.reject(new Error('destinationAccount must be provided'))
     }
 
-    const transferData = {
-      ilp_header: omitUndefined({
-        account: params.destinationAccount,
-        amount: params.destinationAmount,
-        data: params.destinationMemo
-      })
-    }
+    const ilpHeader = packet.serializeIlpPayment({
```

Comment:
Sorry to be a stickler about this but can you change this to be `ilpPayment`? "ILP Header" was the old name but we don't really have the concept of headers in the same way anymore